### DISCUSSION TITLE
Welcome to exchange_calendars Discussions!

### DISCUSSION THREAD

**Discussion by gerrymanoim** [gerrymanoim/exchange_calendars#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Mostly this is a place for `exchange_calendars` things that don't necessarily fit into issues.

**Comment by annisdh** [comment]
Are there plans to add or support Eurex?  I think the ISO 10383 code is ECAG.  Thank you.

**Reply by gerrymanoim** [reply]
All calendars are user contributed.
